### ISSUE TITLE
Debian Package index file parsing problem

### ISSUE THREAD

**Issue by scentoni** [kellyjonbrazil/jc#613]
This Debian Package index file has 20 stanzas, but jc is only returning 2. A little experimentation with modifying the input shows it's specifically returning only the first and last stanza listed in the input.
```
$ curl -sSO 'http://repo.aptly.info/dists/squeeze/main/binary-amd64/Packages'
$ grep Filename Packages |wc -l
      20
$ jc <Packages --pkg-index-deb |jq length                                                       
2
```

**Comment by kellyjonbrazil** [comment]
Hi - thanks for reporting this. I wonder if this has to do with all of the packages having the same name. Hopefully won't be too hard to fix!

**Comment by kellyjonbrazil** [comment]
I have a fix in the `dev` branch that will go in the next release. You can test the new parser by downloading the file below to your parser plugin directory:

https://github.com/kellyjonbrazil/jc/blob/dev/jc/parsers/rpm_qi.py

- Linux/unix: `$HOME/.local/share/jc/jcparsers`
- macOS: `$HOME/Library/Application Support/jc/jcparsers`
- Windows: `$LOCALAPPDATA\jc\jc\jcparsers`

Let me know if you run into any issues!

**Comment by kellyjonbrazil** [comment]
Now available in release v1.25.4
